### PR TITLE
build metasfresh via dockerbuilds

### PR DESCRIPTION
**Pull request by mpeelen**

### CODE REVIEW

File: docker-builds/db/init/init.sh [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+pg_restore -Fc --exit-on-error --username metasfresh --dbname metasfresh /metasfresh_latest.pgdump
+echo "=========="
+echo " ...done!"
+echo "=========="
+
+echo ""
+echo "======================================="
+echo " Applying migrations ..."
+echo "======================================="
+find /docker-entrypoint-initdb.d/migrations -type f -printf '%f\n' | sort | while read f; do
```
Comment (metas-ts) [inline docker-builds/db/init/init.sh:40]: as talked, pls write some quick comments on Y not the migration-tool

File: docker-builds/e2e/compose.yml [hunk 2]
```diff
@@ -0,0 +1,117 @@
+services:
```
Comment (metas-ts) [inline docker-builds/e2e/compose.yml:1]: pls add a readme saying that it runs for 1-2hrs locally and too long at github.
...and also that the tests need maintainance

File: docker-builds/version.info [hunk 3]
```diff
@@ -0,0 +1 @@
+5.176.3
```
Comment (metas-ts) [inline docker-builds/version.info:1]: pls add readme about this

File: backend/de.metas.cucumber/src/test/java/de/metas/cucumber/InfrastructureSupport.java [hunk 4]
```diff
@@ -91,6 +95,24 @@ public void start()
 	{
 		assertThat(started).isFalse(); // guard
 
+        cucumberIsUsingProvidedInfrastructure = StringUtils.toBoolean(System.getenv("CUCUMBER_IS_USING_PROVIDED_INFRASTRUCTURE"), false);
+
+        if (cucumberIsUsingProvidedInfrastructure) {
```
Comment (metas-ts) [inline backend/de.metas.cucumber/src/test/java/de/metas/cucumber/InfrastructureSupport.java:100]:
```suggestion
	// TODO replace runAgainstDockerizedDatabase and cucumberIsUsingProvidedInfrastructure with an enum
        if (cucumberIsUsingProvidedInfrastructure) {
```

File: docker-builds/Dockerfile.cucumber [hunk 5]
```diff
@@ -0,0 +1,25 @@
+ARG REFNAME=local
+FROM metasfresh/metas-mvn-backend:$REFNAME as backend
+
+FROM maven:3.8.4-jdk-8
+
+RUN apt-get update && apt-get install -y locales npm && rm -rf /var/lib/apt/lists/* && localedef -i de_DE -c -f UTF-8 -A /usr/share/locale/locale.alias de_DE.UTF-8
+RUN npm install -g @testmo/testmo-cli
```
[the remark this answers is not in the document]
Comment (mpeelen) [reply docker-builds/Dockerfile.cucumber:7]: yes, nice catch :)

File: docker-builds/db/init/init.sh (hunk 1 above)
Comment (mpeelen) [reply docker-builds/db/init/init.sh:40]: done

File: docker-builds/e2e/compose.yml (hunk 2 above)
Comment (mpeelen) [reply docker-builds/e2e/compose.yml:1]: done

File: docker-builds/version.info (hunk 3 above)
Comment (mpeelen) [reply docker-builds/version.info:1]: done

**Pull request merged**